Source/WebCore/Modules/WebGPU/GPUComputePipeline.cpp

```diff
@@ -42,7 +42,9 @@ void GPUComputePipeline::setLabel(String&& label)
 
 Ref<GPUBindGroupLayout> GPUComputePipeline::getBindGroupLayout(uint32_t index)
 {
-    return GPUBindGroupLayout::create(m_backing->getBindGroupLayout(index));
+    return m_bindGroupLayouts.ensure(index, [this, index] {
```

Comment:
Is this correct? The specification says `A new [GPUBindGroupLayout](https://www.w3.org/TR/webgpu/#gpubindgrouplayout) wrapper is returned each time`:
https://www.w3.org/TR/webgpu/#dom-gpupipelinebase-getbindgrouplayout

Reply:
Yuck, ok. I'll fix this up.

Reply:
Yep, you're right - this code returns the same object each time.